molecule/kind/molecule.yml

```diff
@@ -5,7 +5,7 @@ driver:
   name: delegated
 lint: |
   set -e
-  yamllint -d "{extends: relaxed, rules: {line-length: {max: 120}}}" .
+  yamllint -d "{extends: relaxed, rules: {line-length: {max: 512}}}" .
```

Comment:
For some reason the default line length is broken during scaffolding, so this value never really made any sense. Could probably just remove the line length check but a value of 512 seemed ok at the time.